generics/services/users.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * name : users.js
+ * author : Vishnu
+ * Date : 07-April-2022
+ * Description : All users related api call.
+ */
+
+//dependencies
+const request = require('request');
+const userServiceUrl = process.env.USER_SERVICE_URL;
+const serverTimeout = process.env.SUNBIRD_SERVER_TIMEOUT ? parseInt(process.env.SUNBIRD_SERVER_TIMEOUT) : 5000;
```

Comment:
@VISHNUDAS-tunerlabs - Remove this timeout from env, just get it from the constants file.